### PR TITLE
Run all package-loads under xvfb

### PR DESCRIPTION
**Pull request by timholy**
May fix Gtk and dependent packages (https://github.com/JuliaRegistries/General/pull/6007, https://github.com/JuliaRegistries/General/pull/5890). Hopefully it isn't much of a burden for other packages.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,6 +30,13 @@ notifications:
 os:
   - linux
 
+addons:
+    apt:
+        packages:
+            - xvfb
+            - xauth
+            - libgtk-3-0
```
Comment (DilumAluthge) [inline .travis.yml]: It looks like you are trying to add these dependencies to Travis?

We actually don’t run the auto merge on Travis; we run it on GH Actions.

So you would need to install these dependencies in the GH Actions workflow file here: https://github.com/JuliaRegistries/General/blob/master/.github/workflows/automerge.yml

**Comment by fredrikekre** [comment]
Perhaps make it conditional on if `xvfb` is installed, i.e. something like
```julia
cmd = Cmd(`$(Base.julia_cmd()) -e $(code)`;
          env = Dict("PATH" => ENV["PATH"],
          "PYTHON" => "",
          "JULIA_DEPOT_PATH" => tmp_dir))

if (xvfb = Sys.which("xvfb-run")) !== nothing
    pushfirst!(cmd.exec, xvfb)
end
```
and then we make sure to install it in GH Actions as @DilumAluthge said?

**Comment by timholy** [comment]
Thanks for the suggestions! Both have been implemented.

**Comment by timholy** [comment]
The previous test (https://travis-ci.com/JuliaRegistries/RegistryCI.jl/jobs/262909814) failed in a `key "BCBI_TEST_USER_GITHUB_TOKEN" not found`, I assume this is nothing I need to worry about?

**Comment by fredrikekre** [comment]
> The previous test (https://travis-ci.com/JuliaRegistries/RegistryCI.jl/jobs/262909814) failed in a key "BCBI_TEST_USER_GITHUB_TOKEN" not found, I assume this is nothing I need to worry about?

Likely because secret env vars are only available when the PR is made from a fork in the same repo.

**Comment by DilumAluthge** [comment]
Once https://github.com/JuliaRegistries/RegistryCI.jl/issues/122 is fixed, we’ll be able to run the integration tests on PRs from forks.

**Comment by DilumAluthge** [comment]
But anyway, this PR looks fine to me. Fredrik or anyone feel free to YOLO merge. Or if no one objects after another day or so, I’ll YOLO merge.

**Comment by DilumAluthge** [comment]
@timholy I have deployed this into production. Do you have any Gtk etc. packages to register so that we can see if it works?

**Comment by DilumAluthge** [comment]
Also, it doesn't seem to be causing trouble with any other (e.g. non-GUI) packages.

So all that remains to be seen is if this fixes Gtk and other GUI packages.

**Comment by timholy** [comment]
Yes, I have a pileup of these. But I first need to fix https://github.com/JuliaGraphics/Gtk.jl/pull/460, which is the same as https://github.com/JuliaGizmos/GtkReactive.jl/pull/91, upon which both ProfileView and ImageView depend.

**Comment by DilumAluthge** [comment]
Just FYI. I added a debug statement to see if AutoMerge can find `xvfb-run`. The result:
```
┌ Debug: xvfb: 
│   xvfb = "/usr/bin/xvfb-run"
└ @ RegistryCI.AutoMerge ~/.julia/packages/RegistryCI/MU6OD/src/AutoMerge/guidelines.jl:179
```

So it looks like `xvfb-run` is in the `PATH` on GitHub Actions.

**Comment by timholy** [comment]
It works! https://github.com/JuliaRegistries/General/pull/6407

**Pull request merged**